assets/data_table/main.js

```diff
@@ -422,8 +464,18 @@ function App({ ctx, data }) {
         <div className="navigation__info">
           <h2 className="navigation__name">{data.name}</h2>
           <span className="navigation__details">
-            {totalRows || "?"} entries
+            {totalRows || "?"} {totalRows > 1 ? "entries" : "entry"}
```

Comment:
We also need to handle "0 entries"

```suggestion
            {totalRows || "?"} {totalRows === 1 ? "entry" : "entries"}
```